### PR TITLE
Add webpack and project content

### PR DESCRIPTION
**Pull request by rociac**
The project uses webpack to bundle all the files into main.js. Added tab navigation and content generated and rendered though JavaScript. Looking forward for your feedback!

### CODE REVIEW

**Review by jofer86 — CHANGES_REQUESTED** [review]
## 👋 👋 Hello!! @rociac . I'm the TSE who is going to be reviewing your PR.
#### Let me tell you about how this review is going to be structured. 
- A quick summary of the changes that are to be requested are going to be displayed here.
- The more specific changes are going to be requested with comments directly on your code, so make sure to scroll down to find the specifics.

#### Now that we grounded the basic structure of the review lets get down to business.
- One of the fundamental aspects of this specific project is the use of the module pattern, the module pattern gives our code one key feature, encapsulation, this feature lets us "hide" our code from external tampering leaving only the end result of our code visible. From what I can see in your code is that the module pattern is not there. The content is being generated with constant variables and those being inserted into the code whilst the scope of the project demands that the content is being displayed by a module, as stated [HERE](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) 
image
This means that we should encapsulate every tab content on their own module. Even though the logic for the insertion is right on point we are lacking the encapsulation of our code. Normally for this specific code it will look something like this 
`const contentModule = (() => {
   const renderFunction = () => {
       ...dom manipulation code here } return { renderFunction }} ) ()`
This means that the logic for appending the HTML elements should happen inside the renderFunction and in the end we are only returning the result of that specific function, leaving all of the logic for that magic safely tucked in inside our module.

- I can see that your code is pretty clean, you have a good understanding and use of the ES6 features, so the only change to be requested of your would be that, just the encapsulation and you will be good to go, It's up to you to make your modules in one or multiple files. Please see to make the necessary changes and do ask for another code review. If you are to have questions or need some clarification on the review don't hesitate to contact me.

- No inline comments where necessary as the changes required fall under the general spectrum of the project.

Happy Coding.
-------------------------------------------
#### Jorge Fernando Rincón Figueroa (@jofer86 on Slack)
#### Technical Support Engineer.



`

File: src/app/views/template.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Content from './content';
+
+const Template = (() => {
```
Comment (mcrd25) [inline src/app/views/template.js:3]: This template is fine, but it does not satisfy the requirement of each tab (Contact, Home, Menu) being in it's own module (as seen in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png). The template itself is a module, which is fine, but the content for your tabs should be in their own separate module. 

You will also notice the mention of a Contact and Menu page which Is not clearly implemented/ Does not work as should on your webpage.

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# 

Hi @rociac, 👋 

Great job so far! 👏 I love the name! Lol! Fat food is the best tasting food 😋 

However, your website does not quite yet meet requirements.

### Required Changes
- Each Tab (Contact, Menu, etc.) should have its own module. You can read more about modules from this [link](https://www.theodinproject.com/courses/javascript/lessons/es6-modules) from Odin. As can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page). Please remember that each exported module should be encapsulated into a `div`.
- Your page seems to be a bit wonky. The reservation tab does not change content and the other tab does not load right away for me
- Address any other review comment NOT tagged as [OTPIONAL]

These changes would make your project even better! 😄 
Anything you need, I'd be here!

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by shloch — APPROVED** [review]
hello @rociac

You get a thumbs up from me,
Great work.
See you on next project

**Pull request merged**